config.toml

```diff
@@ -14,3 +14,6 @@ link1_description = "About"
 
 [taxonomies]
   category = "categories"
+
+[markup.goldmark.renderer]
+unsafe = true
```

Comment:
Please add a new line at the end of the config.toml file, and indent the unsafe like category above.